### PR TITLE
multiple pages: format technical tokens

### PR DESCRIPTION
**Pull request by schneiderl**
This PR updates some tokens that had the backticks missing. 
This does not tackle all the issues but it is a head start on fixing this formatting inconsistency. 

It basically updates filenames to have backticks while also fixing some other minor issues found. Please take a look at the changed files to make sure the changes make sense. 

**Also**: I've messed up the commit history but this should not be a problem. The previous commits are already merged so there will be no changes on the master branch (apart from 2 conflicts I've manually fixed).

### CODE REVIEW

**Comment by schneiderl** [comment]
@bl-ue Thank you for your (extremely) quick comment 😄. 

I'm not a git mastermind myself so I thank you for your explanation. My question is: Is it necessary? Can't we just go with what it is? Because as far as I understand those commits will not have any impact on master branch, especially because they will get squashed when we merge.

**Comment by bl-ue** [comment]
I believe you're right.  I don't think we need to do anything.

I actually sent that comment _before_ you made this PR, on the commit itself, becausing I was running a `git pull` to review another PR and I noticed a new branch, so I though I'd head over to that branch to see what you were up to ;P, and then noticed there were some extra commits...so that's why it was so quick. 😊

**Comment by schneiderl** [comment]
@bl-ue It is of great help - and now I know how to properly handle those situations when I mess that up again. 😄

**Comment by bl-ue** [comment]
@schneiderl this is a WIP, right?  If so, you can convert this PR to a [Draft](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/about-pull-requests#draft-pull-requests), will will prevent accidental merges even if it has >2 approvals:
Screen Shot 2021-01-08 at 2 09 45 PM

**Comment by schneiderl** [comment]
When I first opened it - it was not, but I've already found some extra tokens that might need change.

**Comment by schneiderl** [comment]
I have a question about how to treat URLs: Should Urls on description lines be surrounded by backticks? 
@bl-ue @waldyrious @sbrl 

Also, I will finish this PR by updating the style guide on how to properly use backticks on descriptions.

**Comment by bl-ue** [comment]
IMO, no. @schneiderl are you thinking [osx/valet.md](https://github.com/tldr-pages/tldr/blob/master/pages/osx/valet.md)?

**Comment by schneiderl** [comment]
@bl-ue yes, that one for instance. There is some inconsistency on how to treat that. Some pages have it ticked and some don't.
I also think I prefer them not ticked. If there are any more opinions on that I will standardize it.

**Comment by bl-ue** [comment]
Well, in the case of `valet`, I might backtick it because it's not a valid url.  A full URL like http://example.test I probably would not backtick, because then you wouldn't be able to click it in some markdown renderers, like GitHub.

File: pages.pl/common/sudo.md
Comment (bl-ue) [inline pages.pl/common/sudo.md:17]:
```suggestion
- Powtórz ostatnie polecenie poprzedzone `sudo` (tylko w `bash`, `zsh`, etc.):
```

File: pages/common/sudo.md
Comment (bl-ue) [inline pages/common/sudo.md:17]:
```suggestion
- Repeat the last command prefixed with `sudo` (only in `bash`, `zsh`, etc.):
```

**Comment by bl-ue** [comment]
@schneiderl there are some conflicts because of some changes I made&mdash;would you like me to fix them for you?

**Comment by schneiderl** [comment]
@bl-ue Sure 😄 Thank you.

**Comment by bl-ue** [comment]
Fixed.

**Comment by bl-ue** [comment]
@schneiderl while I was fixing the conflicts, I [changed some tokens in the Korean version of `bedtool.md`](https://github.com/tldr-pages/tldr/pull/5119/commits/5dd165b6c10a5595d35f40d24c7baf23f8482d8f#diff-19964cd286607dac19f6ba867a290c9d25a6f104fdcd02fcae5049858f6a6b0d), but I didn't do that for any other translations of `bedtool.md`, and so I would recommend you to do the rest of them.

File: pages.es/common/git-submodule.md
Comment (bl-ue) [inline pages.es/common/git-submodule.md:20]:
```suggestion
`git submodule foreach git pull`
```

**Review by bl-ue — CHANGES_REQUESTED** [review]
Found a `git` → `Git` mistake. 🙂

File: pages/common/kops.md
Comment (sbrl) [inline pages/common/kops.md:14]: The plural doesn't really make sense here.
```suggestion
- Export the cluster configuration to the `~/.kube/config` file:
```

File: pages/common/lpass.md
Comment (sbrl) [inline pages/common/lpass.md:18]: Not sure about backticks here. Also, grammar?
```suggestion
- Generate a new password for gmail.com with the identifier `myinbox` and add to LastPass:
```

File: pages/common/mail.md
Comment (sbrl) [inline pages/common/mail.md:6]: Do we do this for keyboard shortcuts elsewhere already?

File: pages/common/monodevelop.md
Comment (sbrl) [inline pages/common/monodevelop.md:22]: Same here.
```suggestion
- Disable redirection of stdout and stderr to a log file:
```

File: pages/common/mycli.md
Comment (sbrl) [inline pages/common/mycli.md:6]: Same here.
```suggestion
- Connect to a local database on port 3306, using the current user's username:
```

File: pages/common/ngrep.md
Comment (sbrl) [inline pages/common/ngrep.md:14]: and here
```suggestion
- Capture traffic crossing port 22 of interface eth0:
```

File: pages/common/openssl-genrsa.md
Comment (sbrl) [inline pages/common/openssl-genrsa.md:6]: and here
```suggestion
- Generate an RSA private key of 2048 bits to stdout:
```

File: pages/common/opt.md
Comment (sbrl) [inline pages/common/opt.md:10]: An extra dot would aid clarity here.
```suggestion
- Output the Control Flow Graph of a function to a `.dot` file:
```

File: pages/common/parallel.md
Comment (sbrl) [inline pages/common/parallel.md:22]: Same here
```suggestion
- Break stdin into ~1M blocks, feed each block to stdin of new command:
```

File: pages/common/samtools.md
Comment (sbrl) [inline pages/common/samtools.md:10]: same here
```suggestion
- Take input from stdin (-) and print the SAM header and any reads overlapping a specific region to stdout:
```

File: pages/common/sd.md
Comment (sbrl) [inline pages/common/sd.md:13]: Perhaps this will read better?
```suggestion
- Find and replace in a file printing the result to stdout:
```

File: pages/common/secrethub.md
Comment (sbrl) [inline pages/common/secrethub.md:6]: same here
```suggestion
- Print a secret to stdout:
```

File: pages/common/secrethub.md
Comment (sbrl) [inline pages/common/secrethub.md:18]:
```suggestion
- Store a value supplied on stdin as a new or updated secret:
```

File: pages/common/speedtest-cli.md
Comment (sbrl) [inline pages/common/speedtest-cli.md:18]: URLs / domain names don't really work in backticks I don't think.
```suggestion
- Print a list of all speedtest.net servers, sorted by distance, to file:
```

File: pages/common/terraform-fmt.md
Comment (sbrl) [inline pages/common/terraform-fmt.md:18]: Perhaps this will read better?
```suggestion
- Do not list files that were formatted to stdout:
```

File: pages/common/uncrustify.md
Comment (sbrl) [inline pages/common/uncrustify.md:10]:
```suggestion
- Read filenames from stdin, and take backups before writing output back to the original filepaths:
```

File: pages/common/uvicorn.md
Comment (sbrl) [inline pages/common/uvicorn.md:10]: Perhaps this will read better?
```suggestion
- Listen on port 8080 on localhost:
```

File: pages/common/vsce.md
Comment (sbrl) [inline pages/common/vsce.md:18]: Grammar?
```suggestion
- Package the current working directory as a `.vsix` file:
```

File: pages/common/waitress-serve.md
Comment (sbrl) [inline pages/common/waitress-serve.md:9]:
```suggestion
- Listen on port 8080 on localhost:
```

File: pages/linux/ctrlaltdel.md
Comment (sbrl) [inline pages/linux/ctrlaltdel.md:9]: Don't think keyboard shortcuts in descriptions belong in backticks. Also, typo?
```suggestion
- Set CTRL+ALT+DEL to reboot immediately, without any preparation:
```

File: pages/linux/datamash.md
Comment (sbrl) [inline pages/linux/datamash.md:17]: These are strings, so I don't think this is appropriate.
```suggestion
- Get the mean of a single column of numbers ignoring "Na" and "NaN" (literal) strings:
```

File: pages/linux/mimetype.md
Comment (sbrl) [inline pages/linux/mimetype.md:17]:
```suggestion
- Determine the MIME type of stdin (does not check a filename):
```

File: pages/linux/mkfs.vfat.md
Comment (sbrl) [inline pages/linux/mkfs.vfat.md:5]: This is a typo, not something that belongs in backticks.
```suggestion
- Create a vfat filesystem inside partition 1 on device b (`sdb1`):
```

File: pages/linux/rdesktop.md
Comment (sbrl) [inline pages/linux/rdesktop.md:6]:
```suggestion
- Connect to a remote computer (default port is 3389):
```

File: pages/linux/ss.md
Comment (sbrl) [inline pages/linux/ss.md:13]: port numbers again
```suggestion
- Show all TCP sockets connected to the local HTTPS port (443):
```

File: pages/linux/ufw.md
Comment (sbrl) [inline pages/linux/ufw.md:18]: port numbers again
```suggestion
- Allow incoming traffic on port 5432 on this host with a comment identifying the service:
```

File: pages/linux/ufw.md
Comment (sbrl) [inline pages/linux/ufw.md:22]: and again
```suggestion
- Allow only TCP traffic from 192.168.0.4 to any address on this host, on port 22:
```

File: pages/linux/ufw.md
Comment (sbrl) [inline pages/linux/ufw.md:26]: and again
```suggestion
- Deny traffic on port 80 on this host:
```

File: pages/linux/ufw.md
Comment (sbrl) [inline pages/linux/ufw.md:30]: and again
```suggestion
- Deny all UDP traffic to port 22:
```

File: pages/linux/wpa_passphrase.md
Comment (sbrl) [inline pages/linux/wpa_passphrase.md:5]: stdin again. It's a technical term, not a file or command name.
```suggestion
- Compute and display the WPA-PSK key for a given SSID reading the passphrase from stdin:
```

File: pages/osx/duti.md
Comment (sbrl) [inline pages/osx/duti.md:13]: we should be consistent with other URLs here.
```suggestion
- Set Finder as the default handler for the ftp:// URL scheme:
```

File: pages/osx/logger.md
Comment (sbrl) [inline pages/osx/logger.md:13]: port numbers again
```suggestion
- Send the output to a remote syslog server running at a given port. Default port is 514:
```

File: pages/windows/choco-install.md
Comment (sbrl) [inline pages/windows/choco-install.md:14]: This is the name of a file _type_, not a filename in and of itself.
```suggestion
- Install a specific nuspec or nupkg file:
```

**Review by sbrl — CHANGES_REQUESTED** [review]
Oh wow, this was an _enormous_ pull request! Quite possibly the largest I've ever reviewed. I've left precisely 81 comments here - if they don't all show then GitHub wasn't designed for such an enormous review :P

**EDIT:** Nope, it doesn't look like they did show up.

Next time, could you break this PR into chunks please? I see multiple different types of change here - I would suggest they belong in separate PRs.

Summary:

 - Port numbers should not be in backticks.
 - URLs don't belong in backticks either, as it makes them unclickable when the markdown is rendered.
 - Spotted some grammatical errors.
 - stdin, stdout, and stderr are technical terms, not file / device / command names.
 - Commented on some edge cases.

File: pages.fr/common/hping.md
Comment (bl-ue) [reply pages.fr/common/hping.md:15]: @sbrl see #5128.

**Comment by schneiderl** [comment]
@sbrl After your comments on stdin/stdout/stderr: I've removed the backticks from some extra files.
It was not the idea to make this PR so big but it eventually built up to be this monstrosity. 

I'm gonna go through each of your comments and let you know when it is ready. 
When it does, I'm removing this PR from "Drafts".

File: pages/common/awk.md
Comment (schneiderl) [reply pages/common/awk.md:12]: What exactly do you mean by "previous awk example"? I'm unsure I understood.

File: pages.ko/common/bedtools.md
Comment (schneiderl) [reply pages.ko/common/bedtools.md]: removed as it does not make much of a difference to me

**Comment by schneiderl** [comment]
@sbrl Almost all comments are addressed. There is a single question that I've left about a comment that I did not quite understood. 
Would you mind taking a look at it?

File: pages/common/redis-server.md
Comment (sbrl) [inline pages/common/redis-server.md:6]: I think something has gotten confused here. I thought we were removing the `backticks` around port numbers?

File: pages/linux/ctrlaltdel.md
Comment (sbrl) [inline pages/linux/ctrlaltdel.md:13]: Again with the extra backticks here?
```suggestion
- Set CTRL+ALT+DEL to reboot "normally", giving processes a chance to exit first (send SIGINT to PID1):
```

File: pages/linux/ss.md
Comment (sbrl) [inline pages/linux/ss.md:17]: Missed another.
```suggestion
- Show all TCP sockets listening on the local 8080 port:
```

File: pages/linux/ss.md
Comment (sbrl) [inline pages/linux/ss.md:29]: Again, I'm not sure backticks surrounding IP addresses / IP address blocks is a good idea.
```suggestion
- Show all TCP IPv4 sockets locally connected on the subnet 192.168.0.0/16:
```

**Review by sbrl — CHANGES_REQUESTED** [review]
Just a few more comments on some inconsistencies that have popped up.

File: pages/common/redis-server.md
Comment (schneiderl) [reply pages/common/redis-server.md:6]: I'm unsure of what happened but it is good that you've noticed that.

**Comment by schneiderl** [comment]
@sbrl I'm unsure of what happened here. I thought I've already committed all the changes you have requested in your last comment. 
Now it should all be fine.

**Review by sbrl — APPROVED** [review]
Thanks, @schneiderl!

**Comment by schneiderl** [comment]
@bl-ue Is there anything left blocking this from being merged?

For the person who merges this: Please squash the commits.

**Comment by bl-ue** [comment]
@schneiderl No, not that I know of!  Not sure why it says 279 files were modified :smile: maybe merging inconsistencies? I'll merge and squash now!

**Review by bl-ue — APPROVED** [review]

**Comment by bl-ue** [comment]
Finally!

**Pull request merged**
